frontend/src/components/bucket/BucketConfigForm.vue

```diff
@@ -104,27 +104,39 @@ const onCancel = () => {
     >
       <TextInput
         name="bucketName"
-        label="Bucket name (what other users will see)"
+        label="Bucket name *"
+        placeholder="nr-costcontainment-dev"
+        helptext="The name used for the bucket in BCBox's interface."
       />
       <TextInput
         name="bucket"
-        label="Bucket"
+        label="Bucket *"
+        placeholder="bucket0123456789"
+        helptext="A unique identifier for a bucket."
       />
       <TextInput
         name="endpoint"
-        label="Endpoint"
+        label="Endpoint *"
+        placeholder="https://exampleendpoint.gov.bc.ca/"
+        helptext="The URL that specifies how to access the bucket."
       />
       <Password
         name="accessKeyId"
-        label="Access key Identifier"
+        label="Access key identifier / User ID *"
+        placeholder="AKIAIOSFODNN7EXAMPLE"
+        helptext="A public identifier used to access the bucket. Also known as User ID."
       />
       <Password
         name="secretAccessKey"
-        label="Secret access key"
+        label="Secret access key *"
+        placeholder="wjairXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+        helptext="A private key used to authenticate and authorize access to a bucket."
       />
       <TextInput
         name="key"
         label="Key"
+        placeholder="folder/"
```

Comment:
I dont think it needs the final slash. maybethe placeholder should be `eg: 'documents/reports'`

Reply:
As mentioned previously, we need to decide what our general default recommendation behavior should be. it may suffice to just have it be empty and we manipulate the value in service layer in the event the string is empty or undefined, OR we placeholder this to make sure users put in at least a slash or equivalent. Unsure of best course of action.